notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2409 @@
+{
```

Comment:
<p>I think these look great. The first two in particular are very well suited to an "end of chapter" type exercise that could be used as homework. </p><p> </p><p>The last two could maybe be adapted to in-workbook exercises? More of a "check your understanding" type of thing, where the reader could do just a few lines of code that are very similar to what they've just seen. [At least, the "identify" part of the third one could be short, perhaps not the rest.]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>